src/python/repo.py

```diff
@@ -94,6 +94,54 @@ def clone_repo(repo_slug: str, repo_dir: Path) -> git.repo.Repo:
         ) from None
 
 
+@timeout(10 * 60)
+def clone_repo_to_path(repo_name: str, path: str) -> git.repo.Repo:
+    """Clones a repository to a specified path. Alternative to the clone_repo
+    method since it takes a string path and returns a git repo Repo obj.
```

Comment:
`clone_repo` also returns a git repo Repo object, according to its signature on line 61.  So, is the only difference taking a string versus a Path?  If so, then can this method be a two-liner that creates a Path from a string and then calls `clone_repo`?

Reply:
Yes just wrap the string you use into Path(str) that will give you the Path object directly and you can call the original function with that.